src/base/static/scss/_sidebar.scss

```diff
@@ -113,115 +113,14 @@
     }
```

Comment:
FYI ideally I think we should be deleting this file, and decomposing it into the respective components.

Reply:
Yes, good catch.

Reply:
Ah, actually `_sidebar.scss` confusingly has css rules for the *leaflet* sidebar, which has still not been ported. It also has styles for the right sidebar's container, which has also not been ported and will have to wait until we port the `AppView`. So this file has to stay for the time being...

Reply:
That sounds reasonable. Are the new sidebar's styles independent of this file? If so, then we should be able to remove this file once the Leaflet Sidebar and the Right Sidebar's container are ported.

If there is anything that needs to be reused between those components, like variables, then we should be able to share them between components by adding them to a `/src/base/general.scss` file.